src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -227,7 +219,7 @@ public String getTabTitle() {
         if (getBibDatabaseContext().getDatabaseFile() == null) {
             title.append(GUIGlobals.UNTITLED_TITLE);
 
-            if (!database().getEntries().isEmpty()) {
+            if (!getDatabase().getEntries().isEmpty()) {
```

Comment:
Surprisingly, the method is called `hasNoEntries`. Maybe we should rename that to `isEmpty` directly? So we can call `getDatabase().isEmpty()`. @tobiasdiez Could you do this in this PR?